tools/GraphClust/AlignCluster/test-data/alignment1.cons.tsv

```diff
@@ -1,2 +1,2 @@
 	Background model	Columns	Combinations/Pair	Consensus MFE	Covariance contribution	Decision model	Energy contribution	G+C content	Mean pairwise identity	Mean single sequence MFE	Mean z-score	Prediction	Reading direction	SVM RNA-class probability	SVM decision value	Sequences	Shannon entropy	Structure conservation index	alignment	multihit per_species	num_human_seqs_all	num_seqs	num_seqs_all	rscape_#target_E-val	rscape_Found	rscape_MSA	rscape_PPV	rscape_SEN	rscape_TP	rscape_True	rscape_alen	rscape_avgid	rscape_method	rscape_nseq	rscape_out	cluster_bed	cluster_human_ids	cluster_human_loc
-0	dinucleotide	51	2.890	-7.640	-4.520	structural RNA alignment quality	-3.120	0.320	44.050	-5.080	0.220	RNA	forward	1.000	3.530	5.000	1.035	1.500	cluster.aln	0	0	5	12	0.050	5	result.aln_1	100.000	55.560	5	9	50	44.730	GTp	5	RESULTS/result.aln.sum			
```

Comment:
Any idea why this has changed?

Reply:
I think RNAz background model can slightly change, that's RNAz score and not Rscape. And also probability has not changed. So it should be fine.